### PR TITLE
2729 - Material UI - portal

### PR DESCRIPTION
**Pull request by evasileva-epam**

### CODE REVIEW

**Comment by evasileva-epam** [comment]
@AlinaSelyutina, please see updates

File: jdi-light-material-ui-tests/src/main/java/io/github/com/pages/utils/PortalFrame.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.material.elements.inputs.Button;
+import com.epam.jdi.light.material.elements.utils.Portal;
+
+public class PortalFrame extends Section {
+    @UI("//button")
+    public Button button;
+
+    // TODO update with public Label or etc.
+    @UI("//div[@class='makeStyles-alert-1'][1]")
```
Comment (LunAndrey) [inline jdi-light-material-ui-tests/src/main/java/io/github/com/pages/utils/PortalFrame.java:13]: need to change locator to //button/following::div[contains(@class,'jss')][1]

File: jdi-light-material-ui-tests/src/main/java/io/github/com/pages/utils/PortalFrame.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    public Portal field1;
+
+    @UI("//div[@class='makeStyles-alert-1'][2]")
```
Comment (LunAndrey) [inline jdi-light-material-ui-tests/src/main/java/io/github/com/pages/utils/PortalFrame.java:16]: need to change locator to //button/following::div[contains(@Class,'jss')][2]

File: jdi-light-material-ui-tests/src/main/java/io/github/com/pages/utils/PortalFrame.java (hunk 1 above)
Comment (evasileva-epam) [reply jdi-light-material-ui-tests/src/main/java/io/github/com/pages/utils/PortalFrame.java]: I used locator //button/following::div[1] as on 3 different PC there are 3 different class values for field.

File: jdi-light-material-ui-tests/src/main/java/io/github/com/pages/utils/PortalFrame.java (hunk 2 above)
Comment (evasileva-epam) [reply jdi-light-material-ui-tests/src/main/java/io/github/com/pages/utils/PortalFrame.java]: The same as above.

**Pull request merged**